src/utils.js

```diff
@@ -1,17 +1,88 @@
 export function isObject(object) {
-    return typeof object === 'object' && Array.isArray(object) === false;
+  return typeof object === "object" && Array.isArray(object) === false;
 }
 
 export const getMinMax = (arr) => {
-    var max = -Number.MAX_VALUE,
-        min = Number.MAX_VALUE;
-    arr.forEach(function (x) {
-        if (max < x) {
-            max = x;
-        }
-        if (min > x) {
-            min = x;
-        }
-    });
-    return [min, max];
-}
\ No newline at end of file
+  var max = -Number.MAX_VALUE,
+    min = Number.MAX_VALUE;
+  arr.forEach(function (x) {
+    if (max < x) {
+      max = x;
+    }
+    if (min > x) {
+      min = x;
+    }
+  });
+  return [min, max];
+};
+
+/**
+ * Function to convert a hexadecimal color to its RGB equivalent.
+ *
+ * @param {string} hex - The hexadecimal color string. Must start with "#" and be followed by 6 hexadecimal digits.
+ * @returns {Array<number>} An array containing the RGB values (0-255) in the order [r, g, b].
+ */
+export function hexToRGB(hex) {
+  let r = parseInt(hex.slice(1, 3), 16),
+    g = parseInt(hex.slice(3, 5), 16),
+    b = parseInt(hex.slice(5, 7), 16);
+  return [r, g, b];
+}
+
+/**
+ * Function to convert an RGB color to its hexadecimal equivalent.
+ *
+ * @param {Array<number>} rgb - An array of three numbers [r, g, b] representing an RGB color.
+ * @returns {string} A string representing the color in hexadecimal format.
+ */
+export function rgbToHex(rgb) {
+  let r = rgb[0].toString(16),
+    g = rgb[1].toString(16),
+    b = rgb[2].toString(16);
+  return (
+    "#" +
+    ((r.length == 1 ? "0" : "") + r) +
+    ((g.length == 1 ? "0" : "") + g) +
+    ((b.length == 1 ? "0" : "") + b)
+  );
+}
+
```

Comment:
some of these are already available in the base epiviz.gl library - https://github.com/epiviz/epiviz.gl/blob/main/src/epiviz.gl/utilities.js

if its generally useful, lets move some of these utility methods there.

Reply:
Removed this code as it's no longer needed.